sentinelsat/scripts/cli.py

```diff
@@ -212,8 +213,21 @@ def cli(
         search_kwargs.update((x.split("=") for x in query))
 
     if location is not None:
-        wkt, place_name = placename_to_wkt(location)
-        logger.info("The location we are querying is: '%s', '%s'".format(place_name, wkt))
+        wkt, info = placename_to_wkt(location)
+        minX, minY, maxX, maxY = info["bbox"]
+        r = 6371  # average radius, km
```

Comment:
Can you extend the comment how this value is derived and what makes it "average"?

Reply:
At this level of precision (which is more than enough for this simple use case) pretty much all ways of computing the average radius of the Earth round to that value. https://en.wikipedia.org/wiki/Earth_radius